x11-terms/aterm/aterm-1.0.1-r5.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic eutils
+
+DESCRIPTION="rxvt compatible terminal emulator with transparency support"
+HOMEPAGE="http://aterm.sourceforge.net"
+SRC_URI="ftp://ftp.afterstep.org/apps/${PN}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64"
+IUSE="background cjk xgetdefault"
+
+RDEPEND="
+	virtual/jpeg:0
+	media-libs/libpng:0=
+	background? ( media-libs/libafterimage )
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libICE
+"
+DEPEND="${RDEPEND}
+	x11-base/xorg-proto
+	x11-libs/libXt
+"
+
+src_prepare() {
+	# Security bug #219746
+	eapply "${FILESDIR}/${P}-display-security-issue.patch"
+	eapply "${FILESDIR}"/${P}-deadkeys.patch
+	eapply "${FILESDIR}/${P}-dpy.patch"
+	eapply "${FILESDIR}/${P}-remove-streams.patch"
```

Comment:
Lets move these to a `PATCHES` array above the function. E.g.

````
# Security bug #219746
PATCHES=(
	"${FILESDIR}"/${P}-display-security-issue.patch
	"${FILESDIR}"/${P}-deadkeys.patch
	"${FILESDIR}"/${P}-dpy.patch
	"${FILESDIR}"/${P}-remove-streams.patch
)
````

Reply:
Done. But I put quotes around the whole thing rather than just FILESDIR. Should I flip it around? (Just noticed the difference now.)